### DISCUSSION TITLE
"sorry, too many clients already" Error

### DISCUSSION THREAD

**Discussion by madebydor** [nextauthjs/next-auth#9372] [Help]
I'm constantly facing this issue, and I'm not sure if this is related to Next Auth or Postgres, but I'm trying my luck hoping that someone could shed some light about this.
This is a random error I get every few log-ins.
This is the link I'm refered to:
https://next-auth.js.org/errors#adapter_error_getsessionanduser

<img width="958" alt="image" src="https://github.com/nextauthjs/next-auth/assets/7532901/a77c6990-966e-4f66-b70c-0c1abfb6c34c">

**Comment by aryanprince** [comment]
Same issue here, not sure what's causing it.
